tools/cli/commands/connect.py

```diff
@@ -223,6 +227,8 @@ def on_ready():
             print('You can connect to Datalab at ' + datalab_address)
             if not args.no_launch_browser:
                 maybe_open_browser(datalab_address)
+        if CallFlag == True
+        	print("You can connect to Datalab again by passing the command datalab connect <vm-instance-name>"
```

Comment:
This indentation (and the other lines above) have a mix of tabs and spaces, and flakes doesn't like it:

0.71s$ flake8 cli/
cli/commands/connect.py:1:1: E902 TokenError: EOF in multi-line statement
cli/commands/connect.py:160:5: E101 indentation contains mixed spaces and tabs
cli/commands/connect.py:160:5: W191 indentation contains tabs
cli/commands/connect.py:160:14: E225 missing whitespace around operator
cli/commands/connect.py:161:8: E999 SyntaxError: invalid syntax
cli/commands/connect.py:162:5: E101 indentation contains mixed spaces and tabs
cli/commands/connect.py:162:5: W191 indentation contains tabs
cli/commands/connect.py:162:6: E113 unexpected indentation
cli/commands/connect.py:162:14: E225 missing whitespace around operator
cli/commands/connect.py:230:21: E712 comparison to True should be 'if cond is True:' or 'if cond:'
cli/commands/connect.py:231:9: E101 indentation contains mixed spaces and tabs
cli/commands/connect.py:231:9: W191 indentation contains tabs
cli/commands/connect.py:231:80: E501 line too long (107 > 79 characters)